PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/ServerlessSparkSubmissionPanelConfigurable.kt

```diff
@@ -47,6 +47,7 @@ class ServerlessSparkSubmissionPanelConfigurable(private val project: Project, c
                 .doOnEach { submissionPanel.setClustersListRefreshEnabled(true) }
                 .subscribe(
                         { clusters ->
+                            clusters.forEach { cluster -> cluster.getConfigurationInfo() }
```

Comment:
It might be better to do refresh for each cluster in `.fetchClusters()` method with `.flatMap(AzureSparkServerlessCluster::get)`